minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -255,6 +257,11 @@ public void stopWorkers() throws Exception {
       }
     }
     mWorkerThreads.clear();
+
+    // forget all the workers in the master
+    DefaultBlockMaster bm = (DefaultBlockMaster) getLocalAlluxioMaster().getMasterProcess()
+        .getMaster(BlockMaster.class);
+    bm.forgetAllWorkers();
```

Comment:
Should we check if the master is started here/`getMasterProcess()` returns null?

I didn't check the method signature but I think I vaguely remember that it could be possible

Reply:
it checks for `null` now